app/views/admin/shared/_tabbed_navbar.erb

```diff
@@ -0,0 +1,19 @@
+<% nested_menu_items = AdminMenu.nested_menu_items_from_request(request) %>
+
+<% if nested_menu_items[:children].any? %>
+  <header class="mb-3">
+    <h2 class="crayons-title"><%= nested_menu_items[:name].to_s.titleize %></h2>
+    <div class="block s:flex items-center space-between">
+      <nav class="crayons-tabs crayons-tabs--wrapped">
+        <% nested_menu_items[:children].each do |item| %>
+          <a
+            href="/admin/<%= deduced_scope(request) %>/<%= item[:controller] %>"
```

Comment:
is there a chance `deduced_scope()` returns nil?

Reply:
All new admin routes (except overview which is handled differently) will have a the ffg format `/admin/:scope/controller`. The way that this can be nil is if it hits an admin route from the current admin navigation that doesn't follow this format. However,  I've put in "guard clause hack" further above in the admin.html.erb to ensure that the current admin routes do not hit this file.  

I use the word 'hack' for guard clause because its not clean code. Once I move along further I'm going to figure out whether it's worth refactoring it out or I suspect it will not be needed when we remove the current admin navigation.